charts/sonarqube/values.yaml

```diff
@@ -469,8 +471,10 @@ persistence:
   uid: 1000
   guid: 0
 
+  ## DEPRECATED please use root level extraVolumes value
```

Comment:
We should add `(DEPRECATED)` in the readme's descriptions of those parameters together with a small suggestion on what to use instead.